global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "9.0.100-alpha.1.24058.9"
+    "version": "9.0.100-alpha.1.24066.6"
   },
   "tools": {
     "dotnet": "9.0.100-alpha.1.24058.9",
```

Comment:
@martincostello is there any way to configure the bot to update this as well?

Reply:
I can look into making a change so it can do this property too. I'll ping you on the PR once I do the changes to the action.

Reply:
That would be great, thank you!

Reply:
https://github.com/martincostello/update-dotnet-sdk/pull/706

Reply:
@wtgodbe If you want to try updating the SHA to https://github.com/martincostello/update-dotnet-sdk/commits/2e703e86196fd72c4d3b3b4352caac171a3d4314 I think that should sort it for you. If it does, I'll push a new tagged release tomorrow.